pycodestyle.py

```diff
@@ -493,6 +494,17 @@ def whitespace_around_keywords(logical_line):
         elif len(after) > 1:
             yield match.start(2), "E271 multiple spaces after keyword"
 
+    if sys.version_info >= (3, 10):
```

Comment:
do things pass without the conditional?

Reply:
Yeah, it does. The original idea was to not have to check the regex if it's not necessary.

Reply:
🤷 could go either way I guess